packages/preview/chemicoms-paper/0.1.0/typst.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "chemicoms-paper"
+version = "0.1.0"
+entrypoint = "src/lib.typ"
+authors = ["James R. Swift"]
+license = "MIT-0"
+repository = "https://github.com/JamesxX/chemicoms-paper"
+keywords = ["paper", "article", "journal", "conference"]
+categories = ["paper"]
+description = "An RSC-style paper template to publish at conferences and journals"
+exclude = ["README.md", "CHANGELOG.md", "Justfile", "docs", "tests", ".github"]
```

It might be useful to have a copy of the README file in the archive (for instance if a language server wants to use it to show information about your package).

```suggestion
exclude = ["CHANGELOG.md", "Justfile", "docs", "tests", ".github"]
```